### ISSUE TITLE
Use tuple not list for __all__ attributes

### ISSUE THREAD

**Issue by peterjc** [galaxyproject/planemo#607]
TravisCI doctring linting reports loads of warnings like this:

```
planemo/conda.py WARNING: __all__ is defined as a list, this means pydocstyle cannot reliably detect contents of the __all__ variable, because it can be mutated. Change __all__ to be an (immutable) tuple, to remove this warning. Note, pydocstyle uses __all__ to detect which definitions are public, to warn if public definitions are missing docstrings. If __all__ is a (mutable) list, pydocstyle cannot reliably assume its contents. pydocstyle will proceed assuming __all__ is not mutated.
```

Switching ``__all__`` from a list to a tuple ought to fix this?

**Comment by nsoranzo** [comment]
Yes, I did this for the `galaxy` repo.

**Comment by peterjc** [comment]
Did you write a script to do this, and there a too many for me to want to do by hand?

**Comment by nsoranzo** [comment]
No, I didn't, sorry, did it by hand.

**Comment by peterjc** [comment]
I'm working on this now...

**Comment by peterjc** [comment]
Proposed fix added to pull request #606 which was all linting and touched some of the same code so this avoids any merge conflicts.

**Comment by peterjc** [comment]
Fixed via #606, closing
